spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe ReviewsController, :type => :controller do
```

Comment:
I think this should be a nested resource:

I can't remember how to do it off the top of my head though.

Something like ProjectsController::ReviewsController

or something.